client/cmdlf.c

```diff
@@ -687,7 +687,7 @@ int CmdLFpskSim(const char *Cmd)
 	uint8_t carrier=0, clk=0;
 	uint8_t invert=0;
 	bool errors = false;
-	char hexData[32] = {0x00}; // store entered hex data
+	char hexData[64] = {0x00}; // store entered hex data
 	uint8_t data[255] = {0x00}; 
```

Comment:
with hexData at 64 length can it now overflow the data array (64*4bits = 256 bits) ? 
or is the hexData[31] going to be the \0 and get ignored?